graph/types/schema.graphqls

```diff
@@ -51,7 +51,7 @@ input PostContentInput {
 
 
 type Mutation {
-  addDiscussionParticipant(discussionID: String!, userID: String!, discussionParticipantInput: AddDiscussionParticipantInput!): Participant!
+  addDiscussionParticipant(discussionID: String!, discussionParticipantInput: AddDiscussionParticipantInput!): Participant!
```

Comment:
So this endpoint, I think, needs to have userID so the moderator could add a user.